test/cli/config-external/main.js

```diff
@@ -0,0 +1,4 @@
+import { execute } from './_config.js';
```

Comment:
This appears to be [failing on Windows](https://ci.appveyor.com/project/Rich-Harris/rollup/build/611/job/leyq1rf3jk64fp03#L617) – presumably we need to normalise path separators or something?